nifi/base/reencrypt-rbac.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: route-reencrypt-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: route-reencrypt-role
+subjects:
+  - kind: ServiceAccount
+    name: nifi-sa
+    namespace: nifi
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: route-reencrypt-role
+rules:
+  - apiGroups:
+      - ""
+      - route.openshift.io
+    resources:
+      - routes
+      - routes/custom-host
+    verbs:
```

Comment:
You can only keep:
- get
- list
- update
- patch
which will reduce the openings

Reply:
I had made an assumption that replace to use either create or patch but seems that it has its own verb... replace. Going to clean up and resubmit.